core/arch/arm/plat-imx/drivers/imx_csu.c

```diff
@@ -103,6 +103,9 @@ static TEE_Result csu_init(void)
 	const struct csu_config *csu_config = NULL;
 	const struct csu_setting *csu_setting = NULL;
 
+	if (IS_ENABLED(CFG_IMX_INSECURE_CSU_SETTING))
+		IMSG("Device running with an insecure setting, consult the security reference manual from NXP");
```

Comment:
Yes I think there was a consensus. That being said, you could as well print two lines:
```suggestion
	 if (IS_ENABLED(CFG_IMX_INSECURE_CSU_SETTING)) {
		IMSG("Device running with an insecure setting!");
		IMSG("Consult the security reference manual from NXP");
	}
```

Would "*possibly* running" be more accurate?

Reply:
I don't really see how this helps. The embedded core image may contain mis-configured  security elements with or w/o the switch enabled.
From a platform user perspective, I think I would expect platform refuses to boot or provide some services when `CFG_INSECURE_SETTINGS=n` and something is badly configured.